Modules/_io/iobase.c

```diff
@@ -797,11 +806,6 @@ static PyMethodDef iobase_methods[] = {
     _IO__IOBASE_READABLE_METHODDEF
     _IO__IOBASE_WRITABLE_METHODDEF
 
-    {"_checkClosed",   _PyIOBase_check_closed, METH_NOARGS},
```

Comment:
Maybe you forgot to add new implementations of these, I cannot find it anywhere else.

Reply:
Hm, that's possible, but I don't remember; it's been a while since I looked at this.